examples/mix.rs

```diff
@@ -2,12 +2,10 @@
 //!
 //! Serve a single file over TCP
 
-use tokio::io::AsyncWriteExt;
-use tokio::net::TcpListener;
-use tokio_uring::fs::File;
-
 use std::env;
 
+use tokio_uring::{fs::File, net::TcpListener};
+
```

Comment:
Should create a new test ,this test is for tokio type comp consideration.

Reply:
?

Reply:
The mix.rs example jus show that tokio-uring runtime can use the tokio type from tokio crate,such as tokio::net and tokio::spawn.